Zend/tests/compile_time_parent_error_01.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+Using "parent" in class without parent results in compile-time error
+--FILE--
+<?php
+class InvalidClass {
+  function method(parent $p) {}
+}
+
+?>
+--EXPECTF--
+Strict Standards: Cannot use "parent" when no class scope is active in %s on line %d
```

Comment:
This message is wrong, isn't it? Class scope is active, there is just no parent.

Reply:
Oops, I copied the wrong line. Will fix.